NineChronicles.Headless/GraphTypes/ActionQuery.cs

```diff
@@ -182,6 +182,7 @@ public ActionQuery(StandaloneContext standaloneContext)
                             (Dictionary)standaloneContext.BlockChain!.GetState(GoldCurrencyState.Address)
                         ).Currency,
                         CurrencyEnum.CRYSTAL => CrystalCalculator.CRYSTAL,
+                        CurrencyEnum.RUNESTONE => throw new ExecutionError("Unsupported Currency type."),
```

Comment:
It's just a question. If it is not supported, is there a reason to add the `CurrencyEnum.RUNESTONE` now?

Reply:
@moreal I agree. I'll fix it.

Reply:
I still don't know what this pull request achieves. What is the use case of the `CurrencyEnum.RUNESTONE` value? 🤔

Reply:
I was thinking of using it in graphql dealing with currency. It is true that there is no place to use it now. But won't it be possible to use it in the future? Shouldn't `CurrencyEnumType' contain all the currencies defined in lib9c, even if they are not in use right now?
I don't have a strong opinion either, but if I had to define what this change was supposed to accomplish, the questions above would need answers.
If we limit the role of `CurrencyEnumType` to "the query type used", then this pull request should be closed. If we set it to "query type corresponding to the currency type defined in lib9c", we will have to reflect lib9c changes every time.
Right now, I don't think it matters.

Reply:
On the other hand, reflecting lib9c's currency into `CurrencyEnumType` every time doesn't seem to be smooth every time. This is because the number of points to manage increases when the design of the game currency (e.g., ticker) changes.
For that reason, I'm closing this pull request.